lib/services/servertech-obm-service.js

```diff
@@ -23,27 +23,27 @@ function serverTechObmServiceFactory(Promise, BaseObmService, _) {
     util.inherits(ServerTechObmService, BaseObmService);
 
     ServerTechObmService.prototype.reboot = function() {
-        return this._runInternal(this.mibPowerCommand + '.' + this.options.config.port, 'i', '3');
+        return this._runInternal([this.mibPowerCommand + '.' + this.options.config.port, 'i', '3']);
```

Comment:
What's the background of changing it to array, it does't work in old way?

Reply:
ServerTechObmService.prototype._runInternal this function  allow to accept 2 arguments : command and file (file can be null). 
however , "return this._runInternal(this.mibPowerCommand + '.' + this.options.config.port, 'i', '3')" without the brackets actually send 3 arguments to the function ,and it will consider the string "i " as file value, it is not a available snmp command  and will run failed . so we need the brackets to make all the arguments value as the first arguments "command".